### PR TITLE
Activate mnemonic even if it doesn't have a twin id

### PR DESCRIPTION
**Pull request by samaradel**
### Description

- Remove Activation account button
- Trigger the activation while connecting the wallet
- Disable ```E-mail``` field while activating

### Changes

[Screencast from 27-03-24 21:18:00.webm](https://github.com/threefoldtech/tfgrid-sdk-ts/assets/11662805/4b691e4f-2133-4b1c-9990-b241f37d9080)

### Related Issues

- https://github.com/threefoldtech/tfgrid-sdk-ts/issues/2350

### Checklist

- [ ] Tests included
- [x] Build pass
- [ ] Documentation
- [x] Code format and docstrings
- [x] Screenshots/Video attached (needed for UI changes)

### CODE REVIEW

**Review by amiraabouhadid — CHANGES_REQUESTED** [review]
I get this error when I enter the mnemonics of an account that hasnt been activated yet
image

**Comment by samaradel** [comment]
@amiraabouhadid It's deactivated when click connect

**Comment by samaradel** [comment]
@amiraabouhadid error fixed

**Review by zaelgohary — COMMENTED** [review]
Although email is required while activation, it's not in the metadata in the twin details page.

image

**Comment by samaradel** [comment]
@zaelgohary fixed

File: packages/playground/src/weblets/profile_manager.vue [hunk 1]
```diff
@@ -845,10 +812,24 @@ async function storeAndLogin() {
...
-  activate(mnemonic.value, keypairType.value);
+  try {
+    const grid = await getGrid({ mnemonic: mnemonic.value, keypairType: keypairType.value });
+    storeEmail(grid!, email.value);
+    setCredentials(md5(password.value), mnemonicHash, keypairTypeHash, md5(email.value));
+    activate(mnemonic.value, keypairType.value);
+  } catch (e) {
+    if (e instanceof TwinNotExistError) {
+      isNonActiveMnemonic.value = true;
+      isValid.value = true;
```
Comment (zaelgohary) [inline packages/playground/src/weblets/profile_manager.vue:823]: What does this boolean do? Same for the one outside the condition.

**Review by amiraabouhadid — APPROVED** [review]

**Pull request merged**